src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -365,10 +367,34 @@ def reweight(self, svc_id, weight):
     def _create_predefined_drive_group(self, data):
         orch = OrchClient.instance()
         option = OsdDeploymentOptions(data[0]['option'])
+        nvme_model = None
+        ssd_model = None
+        for inventory_host in orch.inventory.list(hosts=None, refresh=True):
+            for device in inventory_host.devices.devices:
+                if device.available and device.human_readable_type == 'ssd':
+                    if '/dev/nvme' in device.path:
+                        nvme_model = device.sys_api['model']
+                    else:
+                        ssd_model = device.sys_api['model']
```

Comment:
This is still a hack... What it more than 1 model is used for SSDs or NVMes? Can't we pass a path pattern to match NMVe/SSD devs?

@adk3798 do you know if we can pass globs to the path of a device?